presidio-image-redactor/app.py

```diff
@@ -47,15 +49,23 @@ def redact():
             """Return a redacted image."""
             params = get_json_data(request.form.get("data"))
             color_fill = color_fill_string_to_value(params)
-            image_file = request.files.get("image")
-            if not image_file:
-                raise InvalidParamException("Invalid parameter, please add image data")
-            im = Image.open(image_file)
-
-            redacted_image = self.engine.redact(im, color_fill)
+            if request.get_json(silent=True) and "image" in request.json:
+                im = Image.open(BytesIO(base64.b64decode(request.json.get("image"))))
+                analyzer_entities = request.json.get('analyzer_entities')
+                redacted_image = self.engine.redact(im, color_fill,
```

Comment:
Please update to allow user to pass in the optional `ocr_kwargs` argument and optional `**text_analyzer_kwargs` (see latest version of [presidio-image-redactor/presidio_image_redactor/image_redactor_engine.py](https://github.com/microsoft/presidio/blob/main/presidio-image-redactor/presidio_image_redactor/image_redactor_engine.py)

Reply:
See below